src/server/bin/kibana.bat

```diff
@@ -9,6 +9,7 @@ set NODE=%DIR%\node\node.exe
 set SERVER=%DIR%\src\bin\kibana.js
 set NODE_ENV="production"
 set CONFIG_PATH=%DIR%\config\kibana.yml
+set NPM=npm
```

Comment:
Looks like you might be signaling to the installer what command to use for `npm install`. Seems like the default is already npm, but I would prefer if we used the npm bundled with the version of node.js installed, or depended on our own version of npm rather than just using whatever has the name in the users path. the npm bundled with node is located at:

```
resolve(dirname(process.execPath), '../lib/node_modules/npm')
```